mbedtls/src/ssl/context.rs

```diff
@@ -260,7 +301,7 @@ impl Drop for Context {
     }
 }
 
-impl Read for Context {
+impl<T: Read> Read for Context<T> {
```

Comment:
Bit of a weird type contraint since we never call read. Either don't add a constraint, or add a constraint on IoCallback. Same for Write.